scripts/webflow-api-sync.ts

```diff
@@ -0,0 +1,180 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import { WebflowClient } from 'webflow-api';
+import yaml from 'js-yaml';
+import { setTimeout } from 'node:timers/promises';
+import util from 'node:util';
+import type { CollectionItem, CollectionItemList } from 'webflow-api/api';
+
+const rateLimitSleep = 1000;
+
+const apiCollectionId = '66f2c79a523a1055b3da7339';
+const apiCategoriesId = '6751672fcfb952c066685384';
+const siteId = '63c092e946f9b71ff6874169';
+const domainIds = ['63d392e62ba46692fb9e082e', '63d392e62ba4666ba69e082d'];
```

Comment:
I assume those are public ids?

Reply:
I wouldn't classify them as secrets since they're just keys within webflow, but I also wouldn't say they're public per-se, since they don't get published anywhere.